projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-results/search-ai-results.component.ts

```diff
@@ -65,7 +69,12 @@ import { ModalAiService } from '../../../../services/modal-ai.service';
     EmptyContentComponent,
     MatCardModule,
     AvatarComponent,
-    MatTooltipModule
+    MatTooltipModule,
+    ToggleIconDirective,
+    CoreModule,
```

Comment:
Is CoreModule required? My IDE does not show any difference without it. If required, do you need whole module or can you import just required thing?